### DISCUSSION TITLE
JSON Edit not show

### DISCUSSION THREAD

**Discussion by sc-hero** [filamentphp/filament#8253] [General]
{"default":"#3d3c3c","primary":"#ff0000","info":"#ff7300","success":"#19c240","warning":"#f0832a","danger":"#ff0000"}

My colors field is json data type,

In filament form,

->schema([
                                ColorPicker::make('colors.default')
                                    ->placeholder("Color Default")
                                    ->filled()
                                    ->required()
                                    ->validationAttribute('Color Default'),
                                ColorPicker::make('colors.primary')
                                    ->placeholder("Color Primary")
                                    ->filled()
                                    ->required()
                                    ->validationAttribute('Color Primary'),
                                ColorPicker::make('colors.info')
                                    ->placeholder("Color Info")
                                    ->filled()
                                    ->required()
                                    ->validationAttribute('Color Info'),
                                ColorPicker::make('colors.success')
                                    ->placeholder("Color Success")
                                    ->filled()
                                    ->required()
                                    ->validationAttribute('Color Success'),
                                ColorPicker::make('colors.warning')
                                    ->placeholder("Color Warning")
                                    ->filled()
                                    ->required()
                                    ->validationAttribute('Color Warning'),
                                ColorPicker::make('colors.danger')
                                    ->placeholder("Color Danger")
                                    ->filled()
                                    ->required()
                                    ->validationAttribute('Color Danger'),
                            ])->columns(2),
                            
                            In edit action, In the color field, not show the old data, how to show json data in each of colors field?Please could you help me?

**Comment by leandrocfe** [comment]
$casts array in your model?
